Header header
float64 altitude
float64 voltage

